lib/avo/fields/belongs_to_field.rb

```diff
@@ -126,6 +133,7 @@ def values_for_type(model = nil)
         end
 
         query.all.limit(Avo.configuration.associations_lookup_list_limit).map do |record|
+          # to_param uses slug so checking primary_key is unnecessary
```

Comment:
```suggestion
```

Comments are important, but I’m not sure how this one is relevant. Do you think it adds value or highlights something worth remembering? If so, could you elaborate on it? Thanks!